playground/src/utils/helper_functions.tsx

```diff
@@ -145,3 +145,24 @@ Did you prove the program first?`;
 
   return checkFields(jsonOutput)
 }
+
+/**
+ * Helper function to format the debug output.
+ */
+export function addNewlineAfterWhitespace(str: string): string {
```

Comment:
question:

- why not return json from the rust side so that js can format as it wants?
- or to format everything as it should in the rust side so that it is in a single place?

Reply:
I have added an issue for that. https://github.com/0xPolygonMiden/examples/issues/89

We should have this consistent with the output of the Run and Proof step. We can always return a JSON from Rust Land. 

I would not do that now because it is unclear which debugging outputs we need exactly. Once we have settled there, I will change it. Agreed?

Reply:
sounds good 👍